domain/build.gradle

```diff
@@ -1,3 +1,7 @@
 plugins {
     id 'org.jetbrains.kotlin.jvm'
 }
+
+dependencies {
+    implementation 'javax.inject:javax.inject:1'
+}
```

Comment:
Tests are not moved?

Reply:
Now I see, have been moved in a separate PR

Reply:
Ok :D